lib/improver/cli/nowcast_optical_flow.py

```diff
@@ -61,10 +60,9 @@ def main(argv=None):
                         "T-2 from which to calculate optical flow velocities. "
                         "The files require a 'time' coordinate on which they "
                         "are sorted, so the order of inputs does not matter.")
-    parser.add_argument("--output_dir", metavar="OUTPUT_DIR", type=str,
-                        default='', help="Directory to write all output files,"
-                        " or only advection velocity components if "
-                        "NOWCAST_FILEPATHS is specified.")
+    parser.add_argument("output_filepath", metavar="OUTPUT_FILEPATH",
```

Comment:
The `Argparser` description needs to be updated to remove the "and (optionally) extrapolate to required lead times".

Reply:
Done